.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: required
 language: java
 jdk:
-  - openjdk7 
```

Comment:
Then we cannot build play... It would be nice to build with Java 8, but run with Java 7, but I don't know how to do that with travis